### DISCUSSION TITLE
Making pull requests for machines correctly

### DISCUSSION THREAD

**Discussion by Tomeister1** [86Box/86Box#4476] [General]
Hey guys unfortunately I made a mistake in trying to make new rom files for lots of OEM machines such as Sony VAIO, IBM Aptiva, Packard bell, Compaq Presario, HP pavilion and many more and am trying to do this the right way so that I can follow the guidelines and try again is there any way I can work on this?

Thank you.

**Comment by jriwanek** [comment]
I suggest starting here, then actually reading our contribution guidlines
https://docs.github.com/en/get-started/start-your-journey/git-and-github-learning-resources

You're also gonna need to learn C so you can add machines to the 86box code.

Please don't keep opening PR's which don't add machines, or follow our contribution guidelines, it's a needless time drain, if you want to *request* a machine to be added, you can open issues or discussions and provide roms.
